src/main/resources/config.yml

```diff
@@ -43,3 +33,33 @@ whitelist:
   # WARNING: the plugin DOES NOT convert your offline UUIDs to online and vice-versa.
   #           You should either stick to one or another, or alternatively .
   uuid-mode: auto
+
+#   When both whitelist and auto-whitelist are enabled, the plugin will automatically
+# add a joining player of selected group to the whitelist for the specified duration.
+#
+#   A single player can only get automatically whitelisted "max-repeats" times.
+# So if it's set to 2, after the first period they will be able to get extra time.
+# * It won't work the second time IF they do not meet current strategy requirements.
+#
+#   Strategies select a group of players who will be automatically whitelisted
+# upon joining the server.
+auto-whitelist:
+  enabled: true
+  #   Any value in range (-∞;0] will be treated as permanent whitelisting,
+  # WHICH SHOULD NOT BE USED (because then what's the point in using a whitelist?)
+  #
+  #   Any value that can not be parsed will result in disabling the automatic
+  # whitelisting feature until the configuration is fixed and reloaded.
+  duration: 7d
+  #   A value of zero will disable the auto whitelisting.
+  #   Any negative number will disable this property and players will be able to
+  # infinitely join the server and get whitelisted every time.
+  max-repeats: 1
```

Comment:
Rename to `limit`, update comments describing what exactly this property limits.